Minor fixes

aburluka opened the pull request.

Commit:
Remove redundant vstorage unmounts

vstorage mount operation makes a daemon and meets a docker pid 1
problem. To minimize zombies process number, provisioner should not
unmount vstorage after every operation
Signed-off-by: Alexander Burluka <aburluka@virtuozzo.com>

avagin commented on vzstorage-pd/virtuozzo-provisioner.go:
```diff
@@ -284,7 +283,7 @@ func (p *vzFSProvisioner) Delete(volume *v1.PersistentVolume) error {
 	master        = flag.String("master", "", "Master URL")
 	kubeconfig    = flag.String("kubeconfig", "", "Absolute path to the kubeconfig")
-	provisionerID = flag.String("name", "", "Unique provisioner name")
+	provisionerID = flag.String("name", "virtuozzo.com/virtuozzo-storage", "Unique provisioner name")
```
provisionerID allows to have a few instances of one provisioner (both instances use the same  provisionerName.
Don't mix these terms. You have to add a separate options for provisionerName.

Commit:
Provisioner name is now cli argument

HA schemes require possibility of creation few instances of provisioner.
So, provisioner ID now desribes instance and provisioner name
specifies k8s name of that provisioner.
Signed-off-by: Alexander Burluka <aburluka@virtuozzo.com>
